client/views/channel/channel.js

```diff
@@ -153,11 +165,15 @@ Channel = BlazeComponent.extendComponent({
  * Scrolls down the page when the user is a at or nearly at the bottom of the page
  */
 var scrollDown = function () {
-  // Check if the innerHeight + the scrollY position is higher than the offsetHeight - 200
-  if ((window.innerHeight + window.scrollY) >= (
-    Number(document.body.offsetHeight) - 200
-    )) {
-    // Scroll down the page
-    window.scrollTo(0, document.body.scrollHeight);
-  }
+  // It has to be in a setTimeout or it won't
+  // scroll all the way down for some reason
+  setTimeout(function () {
```

Comment:
As a matter of fact I used `Meteor.defer()` before I used `Meteor.setTimeout` and also got that error, which is weird, because I've used it in the client before.